components/loadercontainer/overview.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Overview
+page_title: LoaderContainer Overview
+description: Overview of the LoaderContainer for Blazor.
+slug: loadercontainer-overview
+tags: telerik,blazor,loader,container,loadercontainer,overview
+published: True
+position: 0
+---
+
+# LoaderContainer Overview
+
+This article provides information about the LoaderContainer for Blazor and its core features.
+
+The LoaderContainer provides an animated indicator, a panel, and an overlay that can be used when the application is performing a time-consuming operation, for example, loading data.
```

Comment:
Consider adding a sentence about its size and that it takes the size of its parent element and shows above it. Link the other articles with particular features from this paragraph too.